examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/CommissionerDiscoveryDelegateImpl.h

```diff
@@ -57,6 +74,63 @@ class CommissionerDiscoveryDelegateImpl : public chip::Controller::DeviceDiscove
     }
 
 private:
+    static void ReportSleepingCommissioners(chip::System::Layer * _Nonnull aSystemLayer, void * _Nullable context)
+    {
+        ChipLogProgress(AppServer, "CommissionerDiscoveryDelegateImpl().ReportSleepingCommissioners() called");
+        CommissionerDiscoveryDelegateImpl * thiz = (CommissionerDiscoveryDelegateImpl *) context;
+        if (thiz != nullptr && thiz->mCachedTargetVideoPlayerInfos != nullptr) {
```

Comment:
same comments apply here as well as the java impl

Reply:
Added the early return here too.